examples/toolchains/rust/shell.nix

```
@@ -1,3 +1,9 @@
 { pkgs ? import ./nixpkgs.nix { } }:
-
-pkgs.mkShellNoCC { nativeBuildInputs = [ pkgs.bazel_5 ]; }
+with pkgs;
+mkShell {
+  nativeBuildInputs = [ bazel_5 git nix zlib libiconv ]
+    ++ (if stdenv.isDarwin then
+      [ darwin.apple_sdk.frameworks.Security ]
+    else
+      [ ]);
```

You could use `lib.optional` instead:
```suggestion
    ++ (lib.optional stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ]);
```